addons/cppcheckdata.py

```diff
@@ -775,9 +775,9 @@ def __init__(self, filename):
         # root is 'dumps' node, each config has its own 'dump' subnode.
         for cfgnode in data.getroot():
             if cfgnode.tag == 'dump':
-                self.configurations.append(Configuration(cfgnode))
-
-
+                self.configurations.append(cfgnode)
+    def Configuration(self,cfg):
+        return Configuration(cfg)
```

Comment:
```suggestion
        return Configuration(cfg)


```
Keep the two blank lines required by PEP 8 before a function.

Reply:
Thank you for checking. and with this, a few more PEP-8 convention fixes were modified.